Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml

```diff
@@ -104,3 +104,4 @@
   - type: MobPrice
     price: 150
   - type: FloatingVisuals
+  - type: Speech
```

Comment:
What does this do?

Reply:
Needed to make the barks in the first place.